dcicutils/misc_utils.py

```diff
@@ -1349,6 +1349,25 @@ def capitalize1(s):
     return s[:1].upper() + s[1:]
 
 
+"""
+Python's UUID ignores all dashes, whereas Postgres is more strict
+http://www.postgresql.org/docs/9.2/static/datatype-uuid.html
+See also http://www.postgresql.org/docs/9.2/static/datatype-uuid.html
+And, anyway, this pattern is what our portals have been doing
+for quite a while, so it's the most stable choice for us now.
```

Comment:
This is used already in schema validation just as it is. I'm going to leave this precisely how it is because it is what portals are defined to do and the whole point was to borrow the definition they were using for other uses and not to have multiple definitions. if someone reports a bug in what the portals do and we decide to fix it, my code will tolerate the fix since it's operating on the same data.

Reply:
I had already updated the comment when I saw this. I agree it looked confused. Thanks for confirming that. :)